### PR TITLE
Add links to examples to README.rst

### PR DESCRIPTION
**Pull request by bprfh**
Show examples on the first page, so people can easily find them.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -60,7 +60,18 @@ See the output of ``deepspeech -h`` for more information on the use of ``deepspe
 
 **Table of Contents**
 
-
+* `Trying out DeepSpeech with examples <examples/EXAMPLES.rst>`_
```
Comment (lissyx) [inline README.rst]: Looks good, I'm just wondering if it should be at the top. @reuben @kdavis-mozilla  ?

Comment (kdavis-mozilla) [reply README.rst]: Maybe after "Using a Pre-trained Model" but before "Training your own Model"?

Comment (bprfh) [reply README.rst]: I would argue that it is the first item on the list.
If you see a project for the first time, I think a good start is to play around with the examples.


> Maybe after "Using a Pre-trained Model" but before "Training your own Model"?

Looks also good I think.

Comment (lissyx) [reply README.rst]: >  If you see a project for the first time, I think a good start is to play around with the examples.

The problem with this assertion is that it holds depending on context, so we can't satisfy everybody :/

Comment (lissyx) [reply README.rst]: > Maybe after "Using a Pre-trained Model" but before "Training your own Model"?

@bprfh would work for me

**Comment by lissyx** [comment]
Thanks!

**Pull request merged**